vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -175,7 +203,14 @@ def _arrange_page_divs(self, page_divs: _PageDivsType):
         right_main = page_divs["components"]
         right_side = html.Div([right_header, right_main], id="right-side")
 
-        return html.Div([left_side, right_side], id="page-container")
+        collapsable_left_side = dbc.Collapse(
+            children=[left_side],
+            id="collapse",
+            is_open=True,
+            dimension="width",
+            navbar=True,
+        )
+        return html.Div([collapsable_left_side, collapsable_icon, right_side], id="page-container")
```

Comment:
If we could get rid of the dbc.Collapse and instead do it directly via CSS/JS, I think that would be nice given the boostrap theme problem we currently have. If it takes too much time or the same functionality cannot be achieved, then don't worry about it.

I would also add these changes:
`left_side = html.Div([left_sidebar, left_main, collapsable_icon], id="left-side")`

and

`return html.Div([left_side, right_side], id="page-container")`

Reply:
As agreed, first version will be merged with dbc.Collapse component, and I will create ticket for it to be removed in the future.